microsetta_admin/server.py

```diff
@@ -166,11 +168,31 @@ def _translate_nones(a_dict, do_none_to_str):
     return result
 
 
+def _get_projects(include_stats, is_active):
+    projects_uri = API_PROJECTS_URL + f"?include_stats={include_stats}"
+    if is_active is not None:
+        projects_uri += f"&is_active={is_active}"
+    status, projects_output = APIRequest.get(projects_uri)
+
+    if status >= 400:
+        result = {'error_message': "Unable to load project list."}
+    else:
+        cleaned_projects = [_translate_nones(x, True) for x in
+                            projects_output]
+        # if we're not using full project stats, sort
+        # alphabetically by project name
+        if not include_stats:
+            cleaned_projects = sorted(cleaned_projects,
+                                      key=lambda k: k['project_name'])
```

Comment:
This does not need to change, but just as a FYI in case you didn't know about this:

```python
>>> from operator import itemgetter
>>> stuff = [{'bar': 10}, {'bar': 20}, {'bar': 5, 'foo': 15}, {'bar': 16}, {'foo': 3, 'bar': 0}]
>>> sorted(stuff, key=itemgetter('bar'))
[{'foo': 3, 'bar': 0}, {'bar': 5, 'foo': 15}, {'bar': 10}, {'bar': 16}, {'bar': 20}]
>>>
```

...it's one of the nifty builtin methods and works for indexing arrays too. That said, `lambda`s are still extremely readable